auto_dev/commands/run.py

```diff
@@ -164,20 +221,34 @@ def change_directory(self, directory: str) -> None:
 )
 @click.option("-v", "--verbose", is_flag=True, help="Verbose mode.", default=False)
 @click.option("--force", is_flag=True, help="Force overwrite of existing agent", default=False)
+@click.option("--no-fetch", is_flag=True, help="Use local agent package instead of fetching", default=False)
 @click.pass_context
-def run(ctx, agent_public_id: str, verbose: bool, force: bool) -> None:
+def run(ctx, agent_public_id: PublicId, verbose: bool, force: bool, no_fetch: bool) -> None:
     """
-    Run an agent from the local packages registry.
+    Run an agent from the local packages registry or a local path.
 
     Example usage:
-        adev run eightballer/my_agent
+        adev run eightballer/my_agent  # Fetch and run from registry
+        adev run eightballer/my_agent --no-fetch  # Run local agent package named my_agent
     """
     logger = ctx.obj["LOGGER"]
-    logger.info(f"Running agent {agent_public_id}... 🚀")
-    runner = AgentRunner(agent_name=agent_public_id, verbose=verbose, force=force, logger=logger)
+    
+    if no_fetch:
+        logger.info(f"Running local agent package {agent_public_id.name}... 🚀")
+    else:
+        logger.info(f"Fetching agent {agent_public_id}... 🚀")
```

Comment:
Wouldnt this be in the init or strategy? 
Just a bit out of place that logic block especially as it doesnt actually do anything. 
Could just print fetch: bool in the one line